### PR TITLE
JJ's cfp for PDX

### PR DESCRIPTION
**Pull request by jjasghar**
- An ignite
- 2 presentations
- A Bio

Signed-off-by: JJ Asghar <jj@chef.io>

### CODE REVIEW

**Comment by purpleidea** [comment]
@jjasghar I'd recommend splitting this into three PR's as I was asked to do, eg: https://github.com/cfgmgmtcamp/2017-pdx-cfp/pull/5#issuecomment-301294178

HTH

File: ignites/vmware_ignite.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Presentation Title
+VMware ecosystem in 5 minutes
+
+## Abstract
+VMware is everywhere. But most people think it's just ESXi and maybe vCenter on
+top of it. Watching this talk will give you a quick understanding of how
+the VMware ecosystem has grown. Along with the growth, multiple rebranding and
+name changes has caused confusion. If you're boss is thinking about purchasing
+vRA/vRO/NSX/vCAC/vSAN/vCAir for half of your IT budget and you don't know what
+it is, this talk aims to give you clarity here.n
```
Comment (purpleidea) [inline ignites/vmware_ignite.md:10]: .n ?

**Comment by KrisBuytaert** [comment]
@jjasghar  yes please as @purpleidea  mentioned we would like 3 different submissions for all the ignites/ talks .. that way we can vote on them individually.. thnx in advance !

**Comment by jjasghar** [comment]
On it!

**Pull request closed without merging**